cpu/cc26xx_cc13xx/include/cpu_conf_cc26xx_cc13xx.h

```diff
@@ -49,6 +52,483 @@ extern "C" {
 #define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
+/**
+ * @brief   CC26xx/CC13xx specific CPU configuration
+ * @{
+ */
+/**
+ * @brief   This includes the CCFG configuration in the binary for flashing
+ *          onto the micro-controller.
+ */
+#ifndef CONFIG_CC26XX_CC13XX_UPDATE_CCFG
+#define CONFIG_CC26XX_CC13XX_UPDATE_CCFG            0
+#endif
+
+/**
+ * @brief   Force VDDR high setting, enables higher output power but also higher
+ *          power consumption.
+ *
+ * This is also called "boost mode".
+ */
+#ifndef CONFIG_CC26XX_CC13XX_FORCE_VDDR_HH
+#define CONFIG_CC26XX_CC13XX_FORCE_VDDR_HH          0
+#endif
+
+/**
+ * @brief   Disable GPRAM and use 8K VIMS RAM as CACHE RAM.
+ *
+ * @note Disabling CACHE will reduce CPU execution speed (up to 60%).
+ * @note GPRAM is 8KB in size and located at 0x11000000-0x11001FFF if
+ *       enabled.
+ */
+#ifndef CONFIG_CC26XX_CC13XX_DIS_GPRAM
+#define CONFIG_CC26XX_CC13XX_DIS_GPRAM              1
+#endif
```

Comment:
There is a problem when defaulting to true boolean configurations. When Kconfig sets a bool to `n` it is not defined at all. Giving this default as `1` will prevent from ever being able to disable this option from Kconfig. I suggest swapping the logic of this option to `CONFIG_CC26XX_CC13XX_GPRAM`.

Reply:
> it is not defined at all

I mean, the macro produced by Kconfig is not defined at all

Reply:
Thanks for the early catch, didn't know about it.

Changed the logic and nows defaults to `n`.

Off-topic question. Regarding the `CC26XX_CC13XX_BL_PIN`, is there a way to having an undefined value of some sort in Kconfig?

https://github.com/RIOT-OS/RIOT/blob/fb22c6fdfec139badaacf9dd177994544b8bdad0/cpu/cc26xx_cc13xx/Kconfig#L71-L77

Reply:
For example, in there I do use 255 (0xFF), however due to the range it's 0. If I remove the default value it will not be defined at all right?

Reply:
Yeah, that default is not ok, as it is outside the range. I would add a switch like so:
```
config CC26XX_CC13XX_BL_PIN_EN
    bool "Enable bootloader backdoor DIO"

 config CC26XX_CC13XX_BL_PIN
     int "Bootloader backdoor DIO number"
     range 0 31
     depends on CC26XX_CC13XX_BL_PIN_EN
     help 
         DIO (pin) number used to enter the bootloader backdoor at 
         boot time.
```
In that case, if the _EN is not set, the macro is not generated.

Reply:
You could add an additional `CC26XX_CC13XX_BL_PIN_ENABLED`